backend/django/core/templates/projects/list.html

```diff
@@ -7,32 +7,88 @@
   <div class="cardface">
     <h1>Projects</h1>
     <div class="list-group">
-      <table class="table table-striped">
-        <tbody>
-          {% for project in object_list %}
-          <tr>
-            <td>{{ project.name }}</td>
-            <td><a href="{{ project.get_absolute_url }}">Details</a></td>
-            <td><a href="{% url 'projects:project_code' project.pk %}">Annotate</a></td>
-            {% if project|proj_permission_level:request.user.profile > 1 %}
-            <td><a href="{% url 'projects:project_admin' project.pk %}">Admin Dashboard</a></td>
-              {% if project.has_model %}
-                <td><a class="download_btn" name="{{ project.pk }}" onclick="download_model({{project.pk}})">Download Model and Labeled Data</a></td>
-              {% else %}
-                {% if project.labeled_data_count > 0 %}
-                <td><a class="download_btn" name="{{ project.pk }}" onclick="download_data({{project.pk}})">Download Labeled Data</a></td>
-                {% else %}
-                <td><p>No Labeled Data to Download</p></td>
-                {% endif %}
-              {% endif %}
-            {% endif %}
-          </tr>
-          {% endfor %}
-        </tbody>
-      </table>
+      {% for potential_umbrella_project in object_list %}
```

Comment:
So... kinda a dumb question. This is obviously working, but I'm trying to figure out *why* it's working. 

Basically, if there are three projects in folder "testing" then I'm wondering why we don't end up with three "testing" panels with the three projects inside? The outer loop goes over all the projects, so in theory the inner loop should run all three times as well....

Reply:
Yea, you're totally right. Basically, at the bottom I wrote some js in a script to "cleanup" and get rid of any extra panels

Reply:
Oh nevermind. I see you've added a function at the bottom here which removes duplicates